src/Authentication/Authenticators/Session.php

```diff
@@ -698,7 +698,7 @@ private function removeSessionUserInfo(): void
      *
      * @return int|string|null
      */
-    private function getSessionKey(string $key)
+    protected function getSessionKey(string $key)
```

Comment:
It seems the method name is not good.
This method gets the key value in Session User Info.
Better name?

Reply:
Updated to `getSessionUserKey`.